<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>demoExtension</class>
 <widget class="qMRMLWidget" name="demoExtension">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>471</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="inputsCollapsibleButton">
     <property name="text">
      <string>Control Panel</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="1">
       <widget class="QPushButton" name="inferencePushButton">
        <property name="font">
         <font>
          <family>Adobe Devanagari</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Inference</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="outputsCollapsibleButton">
     <property name="text">
      <string>Outputs</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Adobe Devanagari</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Chronic:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Adobe Devanagari</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Acute and Chronic:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Adobe Devanagari</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Central:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Adobe Devanagari</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Leftsided:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Adobe Devanagari</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>rightsided:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>Adobe Devanagari</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>RV/LV&gt;=1:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>Adobe Devanagari</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>RV/LV&lt;1:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <family>Adobe Devanagari</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <family>Adobe Devanagari</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <family>Adobe Devanagari</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_12">
        <property name="font">
         <font>
          <family>Adobe Devanagari</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_13">
        <property name="font">
         <font>
          <family>Adobe Devanagari</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_14">
        <property name="font">
         <font>
          <family>Adobe Devanagari</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_15">
        <property name="font">
         <font>
          <family>Adobe Devanagari</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="label_16">
        <property name="font">
         <font>
          <family>Adobe Devanagari</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_17">
        <property name="font">
         <font>
          <family>Adobe Devanagari</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Indeterminate:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Adobe Devanagari</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Negative Exam for PE:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_18">
        <property name="font">
         <font>
          <family>Adobe Devanagari</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>PE on Image:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLabel" name="label_19">
        <property name="font">
         <font>
          <family>Adobe Devanagari</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_20">
        <property name="font">
         <font>
          <family>Adobe Devanagari</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
